bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -252,14 +275,50 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">Mo
         </div>
       </article>
 
+      <article id="christina-lang" data-id="christina-lang" class="vcard has-bio" itemscope itemtype="http://schema.org/Person">
+        <figure class="headshot">
+          <img class="photo" itemprop="image" alt="" src="{{ static('img/mozorg/about/leadership/christina-lang.jpg') }}">
+          <figcaption><h4 class="fn" itemprop="name">Christina Lang</h4></figcaption>
+        </figure>
+
+        <div class="person-info">
+          <p class="title" itemprop="jobTitle">VP Marketing, North America</p>
+
+          <ul class="utility">
+            <li><a class="link" href="{{ url('mozorg.about.leadership') }}#christina-lang">Link to this bio</a></li>
+          </ul>
+        </div>
+
+        <div class="person-bio">
+          <p>As the VP Marketing, North America, Christina Lang (who prefers to be called Lang) is hyper-focused on creative ways to introduce our suite of products to new North American audiences. Lang joined Mozilla in June of 2021 and brought with her over a decade of expertise in marketing strategy, operations, and program management in the tech space for companies such as Outbrain and Facebook.</p>
+
+          <p>A collaborative problem solver, Lang’s superpower is seeing a path through complexity and bringing everyone along for the ride. She is committed to amplifying the consumer marketing work coming out of Mozilla’s North American Marketing team, and developing efficiencies that will further strengthen the wider organization.</p>
+
+          <p>Prior to Mozilla, she led marketing strategy and operations at Twitch, where she was responsible for optimizing process to support product development and go-to-market strategy, as well as driving annual planning and culture building for the marketing organization.</p>
```

Comment:
**nit (copy consistency):** we use "go to market" in Robin's bio but "go-to-market" in Lang's. Is there a style guide to reference here for consistent hyphen or no hyphen use? (I'm leaning towards hyphenation given current use in [Katherine Bose](https://www.mozilla.org/en-US/about/leadership/#katherine-bose) bio)

Reply:
No style guide I'm aware of, but it's two to one so let's hyphenate.